src/components/Heroes/rankColumns.ts

```diff
@@ -54,11 +54,29 @@ type Row = {
 type Column = ProColumn | PublicColumn | HeroColumn
 
 export const rankColumns = (props: Props) => {
-  return props.tabType === HeroesTab.PRO
-    ? generateProTabColumns(props.strings)
-    : generatePublicTabColumns(props.strings);
+  if (HeroesTab.PRO === props.tabType) return generateProTabColumns(props.strings);
+  if (HeroesTab.PUBLIC === props.tabType) return generatePublicTabColumns(props.strings);
+  return generateTurboTabColumns(props.strings);
 };
 
+const generateTurboTabColumns = (strings: GlobalString) => {
+  const heroColumn = generateHeroColumn(strings);
+
+  const combinedColumns = [heroColumn, {
+    displayName: strings.hero_turbo_pick_rate,
+    field: 'pickRateTurbo',
+    sortFn: true,
```

Comment:
I added a displayFn here which adds one decimal always, e.g. 2.0 instead of 2: https://github.com/odota/web/pull/2702/files#diff-67254100eb551b3ec455e54f473fb9825b9d31a8d9c34cc332e109ce4f3b7ce3R244

Reply:
this is due to the high amount of matches, correct? agreed with the addition.